### ISSUE TITLE
ScaLAPACK error

### ISSUE THREAD

**Issue by Enchufa2** [mpimd-csc/flexiblas#3]
Just to keep track of this (context: https://fedoraproject.org/wiki/Changes/FlexiBLAS_as_BLAS/LAPACK_manager).

ScaLAPACK fails to build against FlexiBLAS in Fedora with the following error:

```
/usr/bin/ld: /tmp/xdsep.uSTGUi.ltrans3.ltrans.o: in function `pdlaed3_':
/builddir/build/BUILD/scalapack-2.1.0/scalapack-2.1.0-mpich/SRC/pdlaed3.f:204:
undefined reference to `dlamc3_'
collect2: error: ld returned 1 exit status
```

Complete build.log here: https://koji.fedoraproject.org/koji/taskinfo?taskID=49164728

**Comment by grisuthedragon** [comment]
Will be fixed in 3.0.3.  Test with plain Scalapack on Ubuntu 20.04 is ok.

**Comment by grisuthedragon** [comment]
Closed in Version 3.0.3
